samples/Vue/basic.vue

```diff
@@ -0,0 +1,21 @@
+<style>
+.red {
+  color: #f00;
+}
+</style>
+
+<template>
+<div>
+  <h2 v-class="red">{{msg}}</h2>
+</div>
+</template>
+
+<script>
+module.exports = {
```

Comment:
It's not strictly required, but most files will include some JavaScript. Even if no `<script>` tag is found, the file still gets compiled into a CommonJS module under the hood (the default `module.exports` will be used).